code/modules/medical/surgery_tools.dm

```diff
@@ -671,8 +676,11 @@ CONTAINS:
 		if (isAI(user) || isintangible(user) || isobserver(user)) return
 		user.lastattacked = src
 		..()
-		if (!defib)
-			src.defib = new /obj/item/robodefibrillator/mounted(src)
+		if(!defib || QDELETED(defib))
+			defib = null // ditch the ref, just in case we're QDEL'd but defib is still holding on
```

Comment:
Extraneous if `!defib == TRUE`?

Reply:
Unfortunately not. In my testing, you could `qdel(defib)` in say, a flockdrone, and `defib` would but not null, hence `|| QDELETED(defib)`.

Reply:
Oh I meant setting `defib` to `null` would be extraneous if `!defib == TRUE`.